### ISSUE TITLE
[Request] add README to autocopied runtime

### ISSUE THREAD

**Issue by sw-yx** [sveltejs/sapper#879]
**Is your feature request related to a problem? Please describe.**

as someone new to sapper, i get scared when i see a bunch of big files in `src/node_modules/@sapper` with no explanation apart from `// This file is generated by Sapper — do not edit it!` on some (not all! weirdly) files

**Describe the solution you'd like**

add a readme to explain what this is, what can be ignored, what files (if any) can be edited, link to more info, and any FAQs (in particular people are probably going to ask about the node_modules thing.. i know i know..)

**Describe alternatives you've considered**

do nothing

**How important is this feature to you?**

moderate importance to demystify sapper file structure for newbies like me

**Comment by sw-yx** [comment]
link to rich's tweet https://twitter.com/rich_harris/status/1139908960520220677?lang=en

**Comment by benmccann** [comment]
It's mentioned in the FAQ now: https://sapper.svelte.dev/faq#why-is-there-a-node_modules-folder-in-src

**Comment by sw-yx** [comment]
sure. i'd still prefer if there were a readme that was shipped with it, but your call, and i wont mind if this is closed.

**Comment by benmccann** [comment]
yeah, I think it's a good idea, which is why I left this open. It'd be easy enough to implement. The README could just point to the FAQ perhaps :slightly_smiling_face:
